apps/judicial-system/web/src/components/InfoCard/useInfoCardItems.tsx

```
@@ -36,42 +35,55 @@ const useInfoCardItems = () => {
+      id: 'defendant-item',
+      title: (
+        <Text variant="h4" as="h4" marginBottom={isMultipleDefendants ? 3 : 2}>
+          {capitalize(
+            isRequestCase(caseType)
+              ? formatMessage(core.defendant, {
+                  suffix: isMultipleDefendants ? 'ar' : 'i',
+                })
+              : isMultipleDefendants
+              ? formatMessage(core.indictmentDefendants)
+              : formatMessage(core.indictmentDefendant, {
+                  gender: defendants && defendants[0].gender,
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)